notebooks/matrix_tube_pipeline_seqcount_norm.ipynb

```diff
@@ -51,10 +56,10 @@
    ]
   },
   {
-   "cell_type": "markdown",
    "metadata": {},
+   "cell_type": "markdown",
    "source": [
-    "# Workflow for normalizing DNA\n",
+    "## Part 1: Workflow for normalizing DNA\n",
```

Comment:
Should we have the total of steps here? I mean something like Part 1/X. 

EDIT: I think having both the parts and the steps will be helpful, sometime like Part 1/4, Step 1/X.

Reply:
Ok, I can add something to make that clearer.  In the long run, my plan is to break this one gigantic notebook into multiple notebooks, for the reasons discussed in FutureTech; once we get there, the different parts will no longer be an issue.